src/asmcnc/skavaUI/screen_probing.py

```diff
@@ -125,7 +125,18 @@ def on_enter(self):
         if self.m.is_spindle_on():
             # Spindle is on, need to turn it off
             log("Spindle is on, turning off before probing")
-            self.m.turn_off_spindle()
+
+            # If parent screen is 'home', spindle should be turned off using the spindle button
+            if self.parent_button.return_screen == 'home':
```

Comment:
I don't like it. The probing screen should not need to know that there might be a blinking spindle button somewhere. It just turns off the spindle in router_machine.
The blinking spindle button should bind to a spindle_on BooleanProperty that is provided by router_machine and turn its blinking on or off accordingly.